apollo-router/src/executable.rs

```diff
@@ -217,32 +217,34 @@ pub async fn rt_main() -> Result<()> {
         _ => {
             return Err(anyhow!(
                 r#"
-    💫 Apollo Router requires a supergraph to be set using '--supergraph':
+===
+APOLLO ROUTER v{whatever}
```

Comment:
Ideally this version number could pop in dynamically?

Reply:
Also related to #793